charts/merlin/values.yaml

```diff
@@ -337,6 +337,7 @@ swagger:
     tag: v3.23.5
   apiHost: "merlin.dev"
   basePath: "/api/merlin/v1"
+  specUrl: "https://raw.githubusercontent.com/gojek/merlin/main/swagger.yaml"
```

Comment:
There might be a case when the swagger spec differs from the Merlin server. For example, we run Merlin 0.15 but swagger spec is still using the `main` branch.

I prefer to keep chart template and update the API to serve swagger.yaml